README.md

```diff
@@ -1 +1,218 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Decode Morse Code] <a name="about-project"></a>
+
+> During your vacation at the seaside, time after time you find old bottles with parchment inside. Out of curiosity, you look into the next bottle and see that there is a message written in Morse code. Your curiosity grows even more - maybe you just found the lost treasure of the pirates! However, you are getting tired of decrypting messages manually. Fortunately, you can program in Ruby and you can help yourself by automating this process.
+
+## 🛠 Built With <a name="built-with">SQL</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Client</summary>
+  <ul>
+    <li><a href="#">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Create_a_method_to_decode_a_Morse_code_character]**
+- **[Create_a_method_to_decode_an_entire_word_in_Morse_code]**
+- **[Create_a_method_to_decode_an_entire_message_in_Morse_code]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming soon](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Project Details 
+
+- Team      
+        - [Zainab Alzubair](https://github.com/Zainab-Alzubair)
+        - [Robert Okoe Alabi](https://github.com/Alabi12)
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+## Clone Git Repository
+
+-Reach to following Github URL:
+```
+https://github.com/Zainab-Alzubair/decode-morse-code
+
+```
+
+-Copy the URL.
+
+-In Visual Studio Code “Terminal” change the current working directory to the location where you want the cloned directory.
+
+-Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+```
+$ git clone {https://github.com/Zainab-Alzubair/decode-morse-code}
+```
+
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need:
+
+- [x] Basic knowledge of **Git**, **GitHub** &**Gitflow**.
+- [x] Basic knowledge of **Ruby** Ruby.
+- [x] Basic knowledge of **VS Code** or any other code-editors.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+    _cd PATH_
+    
+    _git clone https://github.com/Zainab-Alzubair/decode-morse-code.git
+   
+   _cd PATH_
+   
+   _npm install_
+      
+   _npm run build_
+   
+   _npm start_
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
```

Comment:
- [ ] Kindly add commands that are project specific. Running `npm start` on a Ruby code doesn't work.